src/test_l3_dsDisplay.c

```diff
@@ -0,0 +1,435 @@
+    {"dsERR_INVALID_PARAM", (int32_t)dsERR_INVALID_PARAM},
+    {"dsERR_INVALID_STATE", (int32_t)dsERR_INVALID_STATE},
+    {"dsERR_ALREADY_INITIALIZED", (int32_t)dsERR_ALREADY_INITIALIZED},
+    {"dsERR_NOT_INITIALIZED", (int32_t)dsERR_NOT_INITIALIZED},
+    {"dsERR_OPERATION_NOT_SUPPORTED", (int32_t)dsERR_OPERATION_NOT_SUPPORTED},
+    {"dsERR_RESOURCE_NOT_AVAILABLE", (int32_t)dsERR_RESOURCE_NOT_AVAILABLE},
+    {"dsERR_OPERATION_FAILED", (int32_t)dsERR_OPERATION_FAILED},
+    {"dsErr_MAX", (int32_t)dsErr_MAX},
+    {NULL, -1}
+};
+
+/* dsVideoPortType_t */
+const static ut_control_keyStringMapping_t videoPortTypeMappingTable[] = {
+    {"dsVIDEOPORT_TYPE_RF", (int32_t)dsVIDEOPORT_TYPE_RF},
+    {"dsVIDEOPORT_TYPE_BB", (int32_t)dsVIDEOPORT_TYPE_BB},
+    {"dsVIDEOPORT_TYPE_SVIDEO", (int32_t)dsVIDEOPORT_TYPE_SVIDEO},
+    {"dsVIDEOPORT_TYPE_1394", (int32_t)dsVIDEOPORT_TYPE_1394},
+    {"dsVIDEOPORT_TYPE_DVI", (int32_t)dsVIDEOPORT_TYPE_DVI},
+    {"dsVIDEOPORT_TYPE_COMPONENT", (int32_t)dsVIDEOPORT_TYPE_COMPONENT},
+    {"dsVIDEOPORT_TYPE_HDMI", (int32_t)dsVIDEOPORT_TYPE_HDMI},
+    {"dsVIDEOPORT_TYPE_HDMI_INPUT", (int32_t)dsVIDEOPORT_TYPE_HDMI_INPUT},
+    {"dsVIDEOPORT_TYPE_INTERNAL", (int32_t)dsVIDEOPORT_TYPE_INTERNAL},
+    {"dsVIDEOPORT_TYPE_MAX", (int32_t)dsVIDEOPORT_TYPE_MAX},
+    {NULL, -1}
+};
+
+/* dsDisplayEvent_t */
+const static ut_control_keyStringMapping_t displayEventMappingTable[] = {
+    {"dsDISPLAY_EVENT_CONNECTED", (int32_t)dsDISPLAY_EVENT_CONNECTED},
+    {"dsDISPLAY_EVENT_DISCONNECTED", (int32_t)dsDISPLAY_EVENT_DISCONNECTED},
+    {"dsDISPLAY_RXSENSE_ON", (int32_t)dsDISPLAY_RXSENSE_ON},
+    {"dsDISPLAY_RXSENSE_OFF", (int32_t)dsDISPLAY_RXSENSE_OFF},
+    {"dsDISPLAY_HDCPPROTOCOL_CHANGE", (int32_t)dsDISPLAY_HDCPPROTOCOL_CHANGE},
+    {"dsDISPLAY_EVENT_MAX", (int32_t)dsDISPLAY_EVENT_MAX},
+    {NULL, -1}
+};
+
+/* dsVideoAspectRatio_t */
+const static ut_control_keyStringMapping_t dsVideoAspect_RatioMappingTable[] = {
+    {"dsVIDEO_ASPECT_RATIO_4x3", (int32_t)dsVIDEO_ASPECT_RATIO_4x3},
+    {"dsVIDEO_ASPECT_RATIO_16x9", (int32_t)dsVIDEO_ASPECT_RATIO_16x9},
+    {"dsVIDEO_ASPECT_RATIO_MAX", (int32_t)dsVIDEO_ASPECT_RATIO_MAX},
+    {NULL, -1}
+};
+
+/**
+ * @brief Callback function for display events.
+ *
+ * This function is invoked when a display event occurs, providing information about the event type.
+ */
+void DisplayEventCallback(intptr_t handle, dsDisplayEvent_t event, void* eventData)
+{
+    UT_LOG_INFO("DisplayEventCallback dsDisplayEvent_t:[%s] ", UT_Control_GetMapString(displayEventMappingTable, event));
+}
+
+/**
+ * @brief This test initializes the dsDisplay Module.
+ *
+ * This test function initializes the dsDisplay Module.
+ *
+ * **Test Group ID:** 03@n
+ * **Test Case ID:** 001@n
+ *
+ * **Test Procedure:**
+ * Refer to Test specification documentation
+ * [ds-display_L3_Low-Level_TestSpecification.md](../../docs/pages/ds-display_L3_Low-Level_TestSpecification.md)
+ */
+void dsDisplay_Init()
```

Comment:
test_l3_ to the functions (menu functions) names

Reply:
Done